### DISCUSSION TITLE
高德地图都不用台湾省 你们敏感成这个鸟样 怪不得需要维权...

### DISCUSSION THREAD

**Discussion by hktk1024** [WeNeedHome/SummaryOfLoanSuspension#880] [General]
高德地图都直接写台湾而不是台湾省 你们敏感成这个鸟样 怪不得需要维权...

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/109403820/179718815-208bd603-1db7-48c6-bc2d-7ed5f4370001.png)

这边建议全部碾平
